package.json

```diff
@@ -34,29 +34,29 @@
     "url": "https://github.com/solid/solid-panes/issues"
   },
   "homepage": "https://github.com/solid/solid-panes",
-  "dependencies": {
+  "dependenciesFile": {
     "@solid/better-simple-slideshow": "^0.1.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-metalab": "^1.0.0",
     "mime-types": "^2.1.13",
-    "rdflib": ">=0.17.0",
-    "solid-ui": ">=0.11.5"
+    "rdflib": "file://../../linkeddata/rdflib.js",
```

Comment:
The dependenciesFile and dependenciesBackup are just places to keep different versions - because JSON doesn't allow comments :-(   
I find them handy and some other people find them a useful tip. but you can't test them